src/TwigComponent/CHANGELOG.md

```diff
@@ -2,9 +2,10 @@
 
 ## 2.2
 
--   Allow to pass stringable object as non mapped component attribute.
--   Add _embedded_ components.
--   Allow `ExposeInTemplate` to be used on public component methods.
+- Allow to pass stringable object as non mapped component attribute.
+- Add _embedded_ components.
+- Allow `ExposeInTemplate` to be used on public component methods.
+- Add new ComponentRendererInterface, and have the ComponentRenderer autowirable
```

Comment:
```suggestion
- Add new autowireable `ComponentRendererInterface`
```

This will also actually be in version 2.5.

Reply:
Also, this needs to move under a new `## 2.5` header - right now it's listed under 2.2 :)